spec/services/reports/facility_progress_service_spec.rb

```diff
@@ -130,5 +130,27 @@
         end
       end
     end
+
+    it "returns includes data from current day for follow up numbers" do
+      Timecop.freeze do
```

Comment:
Let's freeze this to a predictable date and time, something in the  middle of the day. Spec could fail intermittently around midnight.

I know the window is small in this case, but this has bitten us quite often in the past 😅